data-raw/healthy-places/living-conditions/rough-sleeping.R

```diff
@@ -0,0 +1,95 @@
+# Installing relevant packages
+library(tidyverse)
+library(readxl)
+library(httr)
+library(dplyr)
+library(geographr)
+
+# Extracting relevant data sets from statswalesr
+# library(statswalesr)
+# statswales_search(c("rough sleeping"))
+# statswales_search(c("homelessness"))
+
+# 
+# # Scrape data from stats Wales
+# # Source:https://statswales.gov.wales/v/P2SD
+# df <- statswales_get_dataset("HOUS0454")
+
+# Upload csv files for rough sleepers and population data
+#read.csv()
+#df <- rough_sleepers_data
+#head(population_df)
+#df <- rough_sleepers_data
+
+# Need to filter to only include Welsh data (mid-2023)
+# population_df <- population_df |>
+#   filter(location == "Wales")
+# str(population_df)
+# head(population_df)
+
+population_df_wales <- population_df |>
+  select(`...4`, `...5`, `...6`) |>
```

Comment:
Please add a comment about what these columns represent so that someone looking at the code understands what columns you are selecting.